engine/channels.go

```diff
@@ -148,10 +179,10 @@ func initializeChannelRoleMap() {
 	channelRoleMap[consensusClusterPrefix] = flow.RoleList{flow.RoleCollection}
 }
 
-// IsClusterChannel returns true if channel is cluster-based.
+// ClusterChannel returns true if channel is cluster-based.
```

Comment:
also returns the channel prefix (not just true or false)

Reply:
right, but it already has this explanation in the subsequent lines:
```
// ClusterChannel returns true if channel is cluster-based.
// At the current implementation, only collection nodes are involved in a cluster-based channels.
// If the channel is a cluster-based one, this method also strips off the channel prefix and returns it.
```